product_multi_ean/__manifest__.py

```diff
@@ -1,19 +1,23 @@
-# -*- coding: utf-8 -*-
 # © 2012-2014 Guewen Baconnier (Camptocamp SA)
 # © 2015 Roberto Lizana (Trey)
 # © 2016 Pedro M. Baeza
+# © 2018 Xavier Jimenez (QubiQ)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 {
     'name': 'Multiple EAN13 on products',
-    'version': '8.0.1.0.0',
+    'version': '11.0.1.0.0',
     'license': 'AGPL-3',
     'author': "Camptocamp, "
               "Trey, "
               "Tecnativa, "
+              "QubiQ, "
               "Odoo Community Association (OCA)",
     'category': 'Product Management',
-    'depends': ['product'],
+    'depends': [
+        'sale',
```

Comment:
Instead of adding this dependency, put `compute_sudo=True` and `inverse_sudo=True` and everything will work correctly without adding the permission